regressiontests/gosec_test.go

```diff
@@ -0,0 +1,51 @@
+package regressiontests
+
+import (
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGosec(t *testing.T) {
+	t.Parallel()
+	gopath := os.Getenv("GOPATH")
```

Comment:
Can you please change this to create a new GOPATH in a temporary directory? You can use `gotest.tools/fs` to do this fairly easily. 

You might need to have a GOPATH with two entries - the existing one and the temporary one.